single-interviews.php

```diff
@@ -113,33 +121,41 @@
 				<div id="tables" class="content" style="display: none;">
 					<div class="tabcontent">
 						<ul class="tabnav">
-							<?php if ( $individuals_id ) echo '<li class="active"><a href="#tab1">Individuals<div></div></a></li>'; ?>
```

Comment:
Good catch. I've filed this as #22 and will close it in future work. For now, I'm merging this and will rebase other branches.